messages/de.json

```diff
@@ -0,0 +1,1393 @@
+        "sorcerer": ["Hexer", "Hexer"],
+        "succubus": ["Weiblicher Dämon", "Weibliche Dämonen"],
+        "time lord": ["Zeitherr", "Zeitherren"],
+        "tough wolf": ["Harter Wolf", "Harte Wölfe"],
+        "turncoat": ["Überläufer", "Überläufer"],
+        "traitor": ["Verräter", "Verräter"],
+        "vengeful ghost": ["Rachsüchtiger Geist", "Rachsüchtige Geister"],
+        "vigilante": ["Mitglied der Bürgerwehr", "Mitglierder der Bürgerwehr"],
+        "villager": ["Bürger", "Bürger"],
+        "village drunk": ["Besoffener Bürger", "Besoffener Bürger"],
+        "warlock": ["Zauberer", "Zauberer"],
+        "werecrow": ["Wehrkrähe", "Wehrkrähe"],
+        "werekitten": ["Wehrkatze", "Wehrkatze"],
+        "wild child": ["Wildes Kind", "Wilde Kinder"],
+        "wolf": ["Wolf", "Wölfe"],
+        "wolf cub": ["Wolfsjunge", "Wolfsjungen"],
+        "wolf gunner": ["Wolf Kanonier", "Wolf Kanoniere"],
+        "wolf mystic": ["Wolf Mystiker", "Wolf Mystiker"],
+        "wolf shaman": ["Wolf Shamane", "Wolf Shamanen"],
+        "**": "The following items are names for the 'team reveal' role reveal option",
+        "village member": ["Dorfmitglied", "Dorfmitglieder"],
+        "neutral player": ["Neutraler Spieler", "Neutraler Spieler"],
+        "wolfteam player": ["Wolfteam Spieler", "Wolfteam Spieler"],
+        "***": "The following items are special keys used in player stats",
+        "lover": ["Liebhaber", "Liebhaber"],
+        "vg activated": ["vg activiert", "vgs activiert"],
+        "vg driven off": ["vg abgefahren", "vgs abgefahren"],
+        "entranced": ["verzaubert", "verzaubert"],
+        "****": "The following items are old roles that are no longer in use, but need to be here for legacy reasons",
+        "bureaucrat": ["Bürokrat", "Bürokraten"]
+    },
+    "_gamemodes": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Gamemodes",
+        "aleatoire": "zufällig",
+        "alpha": "Alpha",
+        "boreal": "Nördlich",
+        "charming": "charmant",
+        "classic": "Klassisch",
+        "default": "Standard",
+        "drunkfire": "betrunkenes Feuer",
+        "evilvillage": "böses Dorf",
+        "foolish": "töricht",
+        "guardian": "Wächter",
+        "lycan": "Lykianer",
+        "mad": "Wütend",
+        "maelstrom": "Sog",
+        "mudkip": "Mudkip",
+        "noreveal": "Noreveal",
+        "random": "zufällig",
+        "rapidfire": "Schnellfeuer",
+        "roles": "Rollen",
+        "sleepy": "Schläfrig",
+        "valentines": "Valentinstag",
+        "**": "The following items are old modes that are no longer in use, but need to be here for legacy reasons",
+        "villagergame": "villagergame"
+    },
+    "_role_aliases": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Role_aliases",
+        "crazed shaman": ["cs"],
```

Comment:
These acronyms should match your translated versions or be removed entirely if making an acronym for it doesn't make sense.

You may also wish to add new acronyms for certain roles if the translated name is particularly long.